packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -477,6 +520,22 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
                       canComment={canComment}
                       shouldFocusEditor={shouldFocusCommentEditor}
                     />
+                    {!hideGatingBanner && gatedGroups.length > 0 && (
```

Comment:
Unrealted: based on the memberships we would have to show/hide this which will happen in #5465